File: gestao-atendimento-web/src/app/pages/funcionario/components/funcionario-form/model/funcionario-dto.ts
export interface FuncionarioDto {
  id?: string;
  tipoUsuario: number;
  nome: string;
  cpf: string;
  whatsapp: string;
  email: string;
  senha: string;
  enderecoComercial: EnderecoDto;
}

interface EnderecoDto {
  id?: string;
  rua: string;
  numero: number;
  bairro: string;
  cep: string;
  cidade: CidadeDto;
  estado: EstadoDto;
}

interface CidadeDto {
  id?: string;
  nome: string;
  estado?: string;
}

interface EstadoDto {
  id?: string;
  nome: string;
  sigla: string;
}

File: gestaoAtendimentoApp/src/main/java/com/br/diegoformentin/gestaoatendimentoapp/dto/funcionario/FuncionarioDto.java
package com.br.diegoformentin.gestaoatendimentoapp.dto.funcionario;

import com.br.diegoformentin.gestaoatendimentoapp.dto.EnderecoCadastroDto;
import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class FuncionarioDto {
    private UUID id;
    private Integer tipoUsuario;
    private String nome;
    private String cpf;
    private String whatsapp;
    private String email;
    private String senha;
    private EnderecoCadastroDto enderecoComercial;
}
